_config/passwords.yml

```diff
@@ -2,12 +2,5 @@
 Name: corepasswords
 ---
 SilverStripe\Core\Injector\Injector:
-  SilverStripe\Security\PasswordValidator:
-    properties:
-      MinLength: 8
-      HistoricCount: 6
```

Comment:
These are now set as the default values for the associated config properties on the new `RulesPasswordValidator` class.